checks.d/docker.py

```diff
@@ -456,20 +452,24 @@ def _find_cgroup(self, hierarchy, docker_root):
         # Old cgroup style
         if len(cgroup_mounts) == 1:
             return os.path.join(docker_root, cgroup_mounts[0][1])
+
+        candidate = None
         for _, mountpoint, _, opts, _, _ in cgroup_mounts:
             if hierarchy in opts:
-                return os.path.join(docker_root, mountpoint)
+                if mountpoint.startswith("/host/"):
+                    return os.path.join(docker_root, mountpoint)
+                candidate = mountpoint
+        if candidate is not None:
+            return os.path.join(docker_root, candidate)
+        raise Exception("Can't find mounted %s cgroups." % hierarchy) 
+
 
     def _parse_cgroup_file(self, stat_file):
         """Parses a cgroup pseudo file for key/values."""
-        fp = None
         self.log.debug("Opening cgroup file: %s" % stat_file)
         try:
-            fp = open(stat_file)
-            return dict(map(lambda x: x.split(), fp.read().splitlines()))
+            with open(stat_file, 'r') as fp:
```

Comment:
`'r'` is the default mode in [open](https://docs.python.org/2/library/functions.html#open) method. Nitpick, since it's specified here but not on :445.